apps/aecore/src/aec_blocks.erl

```diff
@@ -146,7 +146,7 @@ serialize_to_map(B = #block{}) ->
      }.
 
 serialize_tx(Tx) ->
-    #{tx => base64:encode(aec_tx_sign:serialize_to_binary(Tx))}.
+    #{<<"tx">> => base64:encode(aec_tx_sign:serialize_to_binary(Tx))}.
```

Comment:
I know addressing of problems should be done in a subsequent PRs but this one was blocking a couple of different tests

Reply:
well done, this is obvious incompatible with deserialize.